rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout, .sibbo-panel"],
+  "detectCmp": [{ "exists": "sibbo-cmp-layout" }],
+  "detectPopup": [{ "visible": ".sibbo-layout" }],
+  "optIn": [{ "click": "#acceptAllMain" }],
+  "optOut": [
+    { "click": ".sibbo-panel__aside__buttons a[data-nav=\"purposes\"]" },
+    { "click": ".sibbo-panel__main__header__actions a[data-focusable=\"reject-all\"]" },
+    { "wait": "500" },
+    { "click": ".sibbo-panel__main__footer__actions a[data-focusable=\"next\"]" },
```

Comment:
On this site, there's only one page: https://www.parador.es/
I'm afraid we'll need to make this rule a bit more complex to handle different cases

Reply:
Rules rewritten to apply conditional branching and thus to include occurrences of 1-page consent banner vs 2-pages ones (intial rules design).

 Note:
- The "If statement" has a selector that only belongs to 1-page banner and appears from page 0 (banner landing interface)
- If wishing to "manually" test that the condition works: `https://www.estadiodeportivo.com/` is a stable example of 1-page consent website. `https://www.todopapas.com/ `(or many others) follows a 2-page consent pattern.